documentation.md

```diff
@@ -0,0 +1,96 @@
+# Typology Processor
+
+## Overview
+
+Here's a brief explanation of each participant:
+
+- Caller: The caller initiates the request to process a transaction.
```

Comment:
The caller in this case is a Rule PRocessor. So if we can note that as the "caller" that would make sense.